### DISCUSSION TITLE
Clarify `time_format` in docs?

### DISCUSSION THREAD

**Discussion by Kyllingene** [xonsh/xonsh#4711] [Ideas]
In the [docs](https://xon.sh/tutorial.html#customizing-the-prompt), `time_string` is glossed over pretty quickly; it took me over an hour to figure out how to even set it, and another 30 minutes trying to figure out how to format it (in this case, a simple link to relevant information would be handy). The main issue is that the environment variable you have to set, `PROMPT_FIELDS`, isn't mentioned in the prompt customization section, since none of the other options require it.

It would simply be nice to have it mentioned that you have to modify `PROMPT_FIELDS` in order to change the `localtime` formatting. That would come with the added benefit of telling users how to add custom fields they can use in their prompt.

**Comment by jnoortheen** [comment]
If possible you can possibly create a PR for that?

**Reply by Kyllingene** [reply]
PR at xonsh/xonsh-docs/pull/5

**Reply by jnoortheen** [reply]
Thanks for taking the effort. Sorry I should have mentioned that the docs are created using sphinx from https://github.com/xonsh/xonsh/blob/main/docs/tutorial.rst 

Please make sure that the file is [valid](https://pypi.org/project/restructuredtext-lint/) after editing.

**Reply by Kyllingene** [reply]
Ah, ok; Will do that then!

**Reply by Kyllingene** [reply]
New PR at #4714

**Reply by jnoortheen** [reply]
great I will check it
